<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CeguiInjectClass</class>
 <widget class="QWidget" name="CeguiInjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>355</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>355</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CeguiInject</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择渲染方式:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_d3d9">
       <property name="text">
        <string>D3D9</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButton_d3d8">
       <property name="text">
        <string>D3D8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>请选择游戏路径:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_path">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>22</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_path">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>21</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>. . .</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_run">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>启动插件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>139</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_install">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>安装插件</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ceguiinject.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_run</sender>
   <signal>clicked()</signal>
   <receiver>CeguiInjectClass</receiver>
   <slot>runGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_path</sender>
   <signal>clicked()</signal>
   <receiver>CeguiInjectClass</receiver>
   <slot>selectPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>runGame()</slot>
  <slot>selectPath()</slot>
 </slots>
</ui>
